### PR TITLE
Updated release workflow

### PR DESCRIPTION
**Pull request by unbohn**
This PR removes the docker image from the release workflow, which will re-enable the automatic PyPi upload.

### CODE REVIEW

**Review by jammont — APPROVED** [review]
Looks good to me 👍

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -5,28 +5,30 @@ on:
...
-        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: |
-        python -m build --sdist --wheel
-        twine upload dist/*
+      - uses: actions/checkout@v3
+
+      - name: Enable ISOFIT Environment
+        shell: bash
+        run: |
+          echo PATH=/home/ray/anaconda3/envs/isofit/bin/:$PATH >> $GITHUB_ENV
```
Comment (pl-kevinwurster) [inline .github/workflows/release.yml:19]: This section should be deleted, right? This job is no longer executing in a Docker container.

Comment (unbohn) [reply .github/workflows/release.yml]: @pl-kevinwurster Good catch! This is of course not needed anymore. Removed in [5a8fb01](https://github.com/isofit/isofit/pull/450/commits/5a8fb01178892be15de73cb38b5a4967df93e197).

**Comment by unbohn** [comment]
@jammont or @pgbrodrick, could one of you merge this in? I'd like to get the 2.10.5 version uploaded to PyPi soon.

**Pull request merged**